seacrowd/sea_datasets/xm3600/xm3600.py

```diff
@@ -0,0 +1,192 @@
+    author = "Thapliyal, Ashish V.  and
+      Pont Tuset, Jordi  and
+      Chen, Xi  and
+      Soricut, Radu",
+    editor = "Goldberg, Yoav  and
+      Kozareva, Zornitsa  and
+      Zhang, Yue",
+    booktitle = "Proceedings of the 2022 Conference on Empirical Methods in Natural Language Processing",
+    month = dec,
+    year = "2022",
+    address = "Abu Dhabi, United Arab Emirates",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.emnlp-main.45",
+    doi = "10.18653/v1/2022.emnlp-main.45",
+    pages = "715--729",
+}
+"""
+
+_DATASETNAME = "xm3600"
+
+_DESCRIPTION = """\
+Crossmodal-3600 dataset (XM3600 in short), a geographically-diverse set of 3600 images annotated with
+human-generated reference captions in 36 languages. The images were selected from across the world,
+covering regions where the languages are spoken, and annotated with captions that achieve consistency in
+terms of style across all languages, while avoiding annotation artifacts due to direct translation.
+The languages covered in the dataset include Filipino, Indonesian, Thai, and Vietnamnese
+"""
+
+_HOMEPAGE = "https://google.github.io/crossmodal-3600/"
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_URLS = {
+    "captions": "https://google.github.io/crossmodal-3600/web-data/captions.zip",
+    "images": "https://open-images-dataset.s3.amazonaws.com/crossmodal-3600/images.tgz",
+    "image_attributions": "https://google.github.io/crossmodal-3600/web-data/image_attributions.csv",
+}
+
+_SUPPORTED_TASKS = [Tasks.IMAGE_CAPTIONING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGS = ["fil", "id", "th", "vi"]
+
+class NewDataset(datasets.GeneratorBasedBuilder):
+    """
+    Crossmodal-3600 dataset (XM3600 in short), a geographically-diverse set of 3600 images annotated with
+    human-generated reference captions in 36 languages. The images were selected from across the world,
+    covering regions where the languages are spoken, and annotated with captions that achieve consistency in
+    terms of style across all languages, while avoiding annotation artifacts due to direct translation.
+    The languages covered in the dataset include Filipino, Indonesian, Thai, and Vietnamnese
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [SEACrowdConfig(name=f"xm3600_{lang}_source", version=datasets.Version(_SOURCE_VERSION), description=f"xm3600_{lang} source schema", schema="source", subset_id=f"xm3600_{lang}",) for lang in _LANGS] + [
+        SEACrowdConfig(
+            name=f"xm3600_{lang}_seacrowd_imtext",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"xm3600_{lang} SEACrowd schema",
+            schema="seacrowd_imtext",
+            subset_id=f"xm3600_{lang}",
+        )
+        for lang in _LANGS
+    ]
+
+    DEFAULT_CONFIG_NAME = "xm3600_id_source"
```

Comment:
Any particular reason why `id` is the default config? We can probably set an arbitrary one by arranging the languages alphabetically and getting the first element:

```suggestion
    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_{sorted(_LANGS)[0]}_source"
```

Reply:
I'm just taking examples at the other dataloaders in the repo, but your solution might work as well.